.github/workflows/dbld-images.yml

```diff
@@ -29,7 +47,7 @@ jobs:
           gh_export CONTAINER_REGISTRY
 
       - name: Build the images
-        run: dbld/rules images -j $(nproc) --keep-going --output-sync
+        run: dbld/rules image-${{ matrix.image }}
```

Comment:
Is not missing the original parameter list: "-j $(nproc) --keep-going --output-sync"

Reply:
Thanks @Kokan for the explanation, I agree, I removed them as they were unnecessary.

Reply:
Thanks for the explanation.